demisto_sdk/commands/download/downloader.py

```diff
@@ -750,520 +863,434 @@ def get_playbook_id_by_playbook_name(self, playbook_name: str) -> Optional[str]:
-        :param action: The action has been made (merge/download)
-        :param file_name: The file name
-        :param file_type: The file type
-        :return: None
-        """
-        logger.info(f'- {action} {file_type} "{file_name}"')
-        if self.run_format:  # TODO: Refactored after format had verbose arg
-            logger.info("")
+        else:  # If the content item doesn't exist in the output pack, create a new directory for it
+            content_item_output_path = output_path / content_item_entity
+            content_item_output_path.mkdir(parents=True, exist_ok=True)  # Create path if it doesn't exist
 
-    def get_corresponding_pack_content_object(
-        self, custom_content_object: dict
-    ) -> dict:
-        """
-        Returns the corresponding pack content object to the given custom content object
-        :param custom_content_object: The custom content object to merge into the pack
-        :return: The corresponding pack content object
-        """
-        file_entity: str = custom_content_object["entity"]
-        file_name: str = custom_content_object["name"]
-        pack_entity_instances: list = self.pack_content[file_entity]
+        shutil.move(src=file_path, dst=content_item_output_path)
 
-        for pack_entity_instance in pack_entity_instances:
-            if file_name in pack_entity_instance:
-                return pack_entity_instance
-        return {}
+        try:  # Clean up temp dir
+            shutil.rmtree(temp_dir, ignore_errors=True)
+        
+        except shutil.Error as e:
+            logger.warning(f"Failed to remove temp dir '{temp_dir}': {e}")
+            logger.debug(traceback.format_exc())
+
+            if self.run_format and content_item_output_path.suffix in (".yml", ".yaml", ".json"):
+                format_manager(
+                    input=str(content_item_output_path),
+                    no_validate=False,
+                    assume_answer=False,
+                )
 
-    @staticmethod
-    def get_corresponding_pack_file_object(
-        searched_basename: str, pack_content_object: dict
-    ) -> dict:
-        """
-        Searches for the file named searched_basename under the pack content object and returns it
-        :param searched_basename: The basename to look for
-        :param pack_content_object: The pack content object
-        :return: The pack file object
-        """
-        for _, file_objects in pack_content_object.items():
-            for file_object in file_objects:
-                if file_object["path"].endswith(searched_basename):
-                    return file_object
-        return {}
+        logger.debug(f"Content item '{content_item_name}' was successfully downloaded.")
+        return True
 
     @staticmethod
-    def update_data(output_path: str, file_path_to_read: str, file_ending: str) -> None:
+    def update_data(file_to_update: Path | str, original_file: str, is_yaml: bool) -> None:
```

Comment:
1. use `Path` for `original_file`
2. change `original_file` to `original_file_path` for clarity

Reply:
`original_file` is directly passed to `tools.get_dict_from_file`, which only accepts a string.
I'll update its name